opennms-container/minion/container-fs/confd/conf.d/org.apache.karaf.management.cfg.toml

```diff
@@ -0,0 +1,9 @@
+[template]
+src = "org.apache.karaf.management.cfg.tmpl"
+dest = "/opt/minion/etc/org.apache.karaf.management.cfg"
+keys = [
+    "/karaf/management/rmi/registry/host",
```

Comment:
This is fine, however we could make the restriction broader to avoid having to update the keys in the future.

Could just use `/karaf/management/rmi` then you will get access to all the children as well.

Same for the other `toml`'s.

Reply:
Good to know, will do this.